feat: CoffeeAGNTCY initial docs (#176)
agntcy/docs: docs/coffee-agntcy/get-started.md

# Get Started with CoffeeAGNTCY

> CoffeeAgntcy is a reference implementation built around a fictitious coffee company to demonstrate how components in the AGNTCY Internet of Agents ecosystem work together.

Reviewer: ```suggestion
CoffeeAGNTCY is a reference implementation built around a fictitious coffee company to demonstrate how components in the AGNTCY Internet of Agents ecosystem work together.
```

This project is meant to exemplify how AGNTCY and open-source agentic standards interoperate through clean, reproducible example code. It is purposefully designed for developers across experience levels, from those taking their first steps in building agentic systems to those with more experience curious about AGNTCY's offerings.

With CoffeeAgntcy, you can:

> * How to leverage the AGNTCY App SDK Factory to write transport and agentic protocol-agnostic clients and server code

Reviewer: ```suggestion
* Learn how to leverage the AGNTCY App SDK Factory to write transport and agentic protocol-agnostic clients and server code.
```

* Explore SLIM and its support for broadcast and unicast messaging.

* Enable observability with the AGNTCY Observe SDK.

> * Learn how to write simple A2A client and server agents

Reviewer: ```suggestion
* Learn how to write simple A2A client and server agents.
```

> * Orchestrate agents using LangGraph via the supervisor-worker agent architecture

Reviewer: ```suggestion
* Orchestrate agents using LangGraph via the supervisor-worker agent architecture.
```

> * Understand how to integrate data sources via MCP

Reviewer: ```suggestion
* Understand how to integrate data sources via MCP.
```

### Corto

Corto functions as a minimal demo application. While Lungo expands to showcase the ever-growing ecosystem of agentic tooling, Corto remains intentionally simple, serving as an accessible entry point for developers new to the agentic world or to AGNTCY.

Its role is that of a coffee sommelier: you can ask questions like “What does coffee harvested in Colombia in the summer taste like?” and Corto will respond with a clear, engaging description of its flavor profile.

With just two agents, Corto demonstrates the core patterns of agent-to-agent interaction. A LangGraph-orchestrated supervisor agent acts as an A2A client, connecting to a Q Grader Agent Sommelier (A2A server) over a SLIM transport connection. AGNTCY’s observability tools are integrated via the SDK to provide an end-to-end view.

```mermaid
flowchart TB
    A["Supervisor Agent<br/>Buyer"]
    T(["A2A : SLIM"])
    B["Q Grader Agent<br/>Sommelier"]

    A <--> T
    T <--> B
```

Learn more on how to deploy Corto locally by visiting the [Corto deployment guide](https://github.com/agntcy/coffeeAgntcy/blob/main/coffeeAGNTCY/coffee_agents/corto/README.md)

### Lungo

> Lungo is our evolving demo application. As AGNTCY expands, Lungo grows alongside it, adding new features and capabilities, and demonstrating how they work together in an interoperable ecosystem. Like the Corto demo, it includes a LangGraph-orchestrated supervisor agent, but instead of connecting to a single farm, Lungo integrates with three.

Reviewer: ```suggestion
Lungo is our ever-evolving demo application. As AGNTCY expands, Lungo grows alongside it. It adds new features and capabilities, demonstrating how they work together in an interoperable ecosystem. Like the Corto demo, it includes a LangGraph-orchestrated supervisor agent, but instead of connecting to a single farm, Lungo integrates with three.
```

# Get Started with CoffeeAGNTCY

CoffeeAgntcy is a reference implementation built around a fictitious coffee company to demonstrate how components in the AGNTCY Internet of Agents ecosystem work together.

This project is meant to exemplify how AGNTCY and open-source agentic standards interoperate through clean, reproducible example code. It is purposefully designed for developers across experience levels, from those taking their first steps in building agentic systems to those with more experience curious about AGNTCY's offerings.

With CoffeeAgntcy, you can:

* Explore SLIM and its support for broadcast and unicast messaging.

* Enable observability with the AGNTCY Observe SDK.

* Learn how to write simple A2A client and server agents

* Orchestrate agents using LangGraph via the supervisor-worker agent architecture

* Understand how to integrate data sources via MCP

> * How to leverage the AGNTCY App SDK Factory to write transport and agentic protocol-agnostic clients and server code

Reviewer 2: move this first in the list
